src/demo/rid3/demo.cljs

```diff
@@ -133,7 +133,7 @@
                 client-width (some-> js/document .-body .-clientWidth)
                 page-width   (or inner-width client-width)]
 
-            (swap! app-cursor  assoc
+            (swap! ratom  assoc
```

Comment:
This change will actually place `:page-width` and `:mobile?` at the wrong level in the `app-state` ratom. The `app-cursor` is one level nested inside of `app-state` (inside the `:app`) key.

If you wanted to update this PR to create the cursor from the function argument, you could update it like this
```
(swap! (reagent/cursor ratom [:app])  assoc
     ...
```

Reply:
We could use the already defined app-cursor when calling window-size.